app/models/puget_sound_custom_alert_feed.rb

```diff
@@ -0,0 +1,18 @@
+class PugetSoundCustomAlertFeed < AlertFeed
```

Comment:
This is good - I wonder if `PugetSoundManualAlertFeed` would be more explicit? Hard to say.